meteor/client/lib/editAttribute/EditAttributeTextDropdown.tsx

```diff
@@ -1,5 +1,5 @@
-import { IEditAttributeBaseProps } from './edit-attribute-base'
-import { DropdownOption, EditAttributeDropdown } from './edit-attribute-dropdown'
+import { IEditAttributeBaseProps } from './EditAttributeBase'
```

Comment:
What is the reason that we change the filenames format from lowercased kebab-casing to Pascal-casing?

Reply:
Consistency in the repo. Those file names were standing out.